src/components/picker/picker.tsx

```diff
@@ -195,28 +197,30 @@ export class Picker {
             props.maxItems = 1;
         }
 
-        return [
-            <limel-chip-set
-                type="input"
-                label={this.label}
-                leadingIcon={this.leadingIcon}
-                value={this.chips}
-                disabled={this.disabled}
-                readonly={this.readonly}
-                required={this.required}
-                searchLabel={this.searchLabel}
-                onInput={this.handleTextInput}
-                onKeyDown={this.handleInputKeyDown}
-                onChange={this.handleChange}
-                onInteract={this.handleInteract}
-                onStartEdit={this.handleInputFieldFocus}
-                onStopEdit={this.handleStopEditAndBlur}
-                emptyInputOnBlur={false}
-                clearAllButton={this.multiple}
-                {...props}
-            />,
-            this.renderDropdown(),
-        ];
+        return (
+            <Host tabIndex={0}>
```

Comment:
This makes it possible to tab to this component

Reply:
This must be why we get the blue outlines around the component while tabbing, right?

https://user-images.githubusercontent.com/35954987/128882867-92024a0c-0c62-480a-a8fb-76bd7921697a.mov

I'm not sure if it's a good/intended behaviour. I think tabbing should be generally improved for the pickers. Even before your changes, user firs tabs into the chip(s), and then tabs to the input. 

I believe this is the correct order for tab index:
1. **input** should be the first thing users tab into. From here, users should be able to go through the search/suggestions list. But I think that is expected to be done via Enter, then arrow keys up and down. OR(?)
2. then *chips* individually (where they can also navigate left and right with keyboard, or press backspace to delete one.


And the main container component shouldn't receive focus individually (the blue outlines), because users cannot do anything there while the blue lines are shown anyway. They must tab once more to go deeper.

Reply:
The worst part is, when user actually has focused on the input field (when the green outlines are shown), they can't do anything by pressing the enter key. They cannot for example activate the search results or suggestions list.

Reply:
> This must be why we get the blue outlines around the component while tabbing, right?

Exactly!

Reply:
Solved ⚡

Reply:
Yes! And can't find a good solution to that. I'm leaning towards creating a new issue for that. I would argue its still better than before. Right? 🙈

Reply:
Yes, the whole is better than before I'd say. :) 
But it's important to make an issue from my comment above, so we don't forget it. 

They can still click outside to close that list, but I don't know how annoying it is for people who are dependent on keyboard navigation.

Reply:
Was this resolved? If not, should we create an issue, or just react if we hear complaints about it?

Reply:
yes it was fixed ;)